k4FWCore/src/PodioDataSvc.cpp

```diff
@@ -62,6 +63,22 @@ StatusCode PodioDataSvc::initialize() {
     m_metadataframe = podio::Frame();
   }
 
+  IProperty* property;
+  auto       sc = service("ApplicationMgr", property);
+  if (sc == StatusCode::FAILURE) {
+    error() << "Could not get ApplicationMgr properties" << std::endl;
+  }
+  Gaudi::Property<int> evtMax;
+  evtMax.assign(property->getProperty("EvtMax"));
+  m_requestedEventMax = evtMax;
```

Comment:
```suggestion
  m_requestedEventMax = std::stoi(property->getProperty("EvtMax").toString());
```
I liked the one liner